web/templates/modal_banner/message_moving_warning_banner.hbs

```diff
@@ -0,0 +1,7 @@
+{{#> modal_banner }}
+    <p class="banner_message">
+        {{#tr}}
+            You are moving messages to a topic that already exists. Messages from these topics will be combined.
```

Comment:
Is this the intended text? I think the other modal mentions it is not reversible.

Reply:
Has this been discussed on CZO? If not, it would probably be good to start a discussion on it in design or frontend.

Reply:
This is the [CZO thread](https://chat.zulip.org/#narrow/stream/9-issues/topic/.F0.9F.93.81.20.22Merge.20with.20another.20topic.3F.22.20dialog.20.2F.20banner/near/1883678)